packages/ui/nuxt.config.js

```diff
@@ -179,6 +165,22 @@ export default {
     // https://go.nuxtjs.dev/stylelint
     '@nuxtjs/stylelint-module',
     '@nuxtjs/svg',
+    [
+      '@nuxtjs/google-fonts',
+      {
+        families: {
+          'Roboto+Mono': [400],
+          Almarai: [300, 400, 600],
+        },
+        subsets: ['latin'],
```

Comment:
I think `latin` is always loaded by default: https://developers.google.com/fonts/docs/getting_started#specifying_script_subsets.

Also, why if we only specified `latin` we have more subsets in Roboto
[packages/ui/assets/css/fonts.css](https://github.com/witnet/data-feeds-explorer/pull/162/files#diff-926fd0fe77f8d2edeb36cc5c76ea1ffa9df2ef69bacd54e6457ecc7215c3497c) but none for Almara?